src/App.js

```diff
@@ -1,79 +1,15 @@
 import React from 'react';
 import './App.css';
+import Welcome from './components/welcome/welcome';
+import Header from './components/header/header';
+import Article from './components/article/article';
```

Comment:
```suggestion
import Welcome from './components/Welcome/Welcome';
import Header from './components/Header/Header';
import Article from './components/Article/Article';
```
компоненты называются с большой буквы